ui/src/components/ClientScopesTable/index.jsx

```diff
@@ -113,8 +114,9 @@ export default class ClientScopesTable extends Component {
       map(node => (
         <TableRow key={node}>
           <TableCell padding="dense">
-            <Link
-              className={classes.tableCell}
+            <TableCellItem
+              button
+              component={Link}
```

Comment:
I think TableCellItem accepts a `dense ` prop to make the items more compact. Let's give this a try. You could have a look at https://taskcluster-ui.herokuapp.com/tasks/index. I think the implementation and look should be similar.